.prettierrc

```diff
@@ -1,5 +1,13 @@
 {
   "singleQuote": true,
   "trailingComma": "all",
-  "printWidth": 100
+  "printWidth": 100,
+  "overrides": [
+    {
+      "files": "*.svelte",
+      "options": {
+        "plugins": ["prettier-plugin-svelte"]
```

Comment:
If I create a new SvelteKit project with create-svelte then plugins is at the top level and I also have a parser override

Reply:
Oh I didn't know about that, I just thought of adding a plugin for just .svelte files. if the global plugin works, its fine ig 😅

Reply:
I'd be curious if one is better for performance or if there is a best practice. Like is yours fast because it applies only to svelte files or is the create-svelte one faster because it just has to be instantiated once globally. The more surprising part is that you were able to get away with out the `parser` setting.

For reference, here's what `create-svelte` does:
```
	"plugins": ["prettier-plugin-svelte"],
	"overrides": [{ "files": "*.svelte", "options": { "parser": "svelte" } }]
```

Reply:
That's weird, parser svelte shouldn't be needed, I have never added that to any prettierrc file until now.

I tested with `pnpx prettier . --write` to see if maybe parser svelte is required, but no, it works just as well